server/src/test/proto/spine/test/bus/bus.proto

```diff
@@ -0,0 +1,58 @@
+//
+// Copyright 2018, TeamDev Ltd. All rights reserved.
+//
+// Redistribution and use in source and/or binary forms, with or without
+// modification, must retain the above copyright notice and the following
+// disclaimer.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+// "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+// LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+// OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+// LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+// DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+// THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+syntax = "proto3";
+
+package spine.test.bus;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine. io";
+option java_package = "io.spine.test.bus";
+option java_outer_classname = "BusProto";
+option java_multiple_files = true;
+
+import "google/protobuf/any.proto";
+
+import "spine/core/user_id.proto";
+import "spine/net/email_address.proto";
+import "spine/test/event/project.proto";
+import "spine/test/event/team.proto";
+
+message BusMessageId {
+    string value = 1;
+}
+
+message BusMessage {
+    
+    BusMessageId id = 1 [(required) = true];
+    
+    google.protobuf.Any contents = 2 [(required) = true];
+    
+    BusMessageContext context = 3;
+    
+}
+
+message BusMessageContext {
```

Comment:
Please fill with the real attributes. Consider those defined in `CommandContext`, `EventContext` or `RejectionContext`.